modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form-report/components/list/List.js

```diff
@@ -28,13 +30,27 @@ export default function List({data, field, summary, totalEntries, type}) {
 
 		return moment(field).locale(locale).format('L');
 	};
+	const formatDateTime = (field) => {
+		const locale = themeDisplay.getLanguageId().split('_', 1).join('');
+
+		const date = moment(field).locale(locale).format('L');
+		const time = moment(field).locale(locale).format('LT');
+
+		const [hourFormat] = time.split(NOT_DIGIT_REGEX, 1);
+
+		const formattedTime = hourFormat.length === 1 ? `0${time}` : time;
+
+		return `${date} ${formattedTime}`;
```

Comment:
I might be wrong, but I see no reasons to change the actual format, cause it's just a view and should not impact how the server is storing the information.  
```suggestion
		return `${date} ${time}`;
```

Reply:
I previously thought this was a requirement for this feature but as you explained I realised it doesn't seem to be; I will change that, thanks!